### ISSUE TITLE
Improve model readme md templates

### ISSUE THREAD

**Issue by marnovo** [src-d/models#2]
Some to-dos:
- Add more information about `src-d/modelforge`'s workings and relationship w/ the models.
- Add file size information.
- Add more structure to the descriptions.
- Add more information about the process of training the models, a more detailed overview, etc
Example that could be used as a template (more suggestions welcome), being more structured and informative: https://nlp.stanford.edu/projects/glove/

**Comment by marnovo** [comment]
@EgorBu feel free to add suggestions (and help, heh).

**Comment by vmarkovtsev** [comment]
@marnovo What is more structure in the descriptions?

The process of training the models is deprecated and does not use UAST and Babelfish for example. In fact, they are hacky converted proof-of-concept models which only demonstrate that "something works" without any guarantee of the quality. I feel that I should not explain the bloody hell which born them.

**Comment by marnovo** [comment]
Sorry, being more specific: structure = organize the descriptions setting headings, lists with the content elements to make it easier to consume the information and to navigate/compare models.

**Comment by r0mainK** [comment]
@marnovo is this still relevant or ?

**Comment by marnovo** [comment]
@r0mainK yes—it marginally improved from [the baseline](https://github.com/src-d/models/tree/HEAD@%7B2017-10-16T15:31:13Z%7D), but still light years behind [the benchmark](https://nlp.stanford.edu/projects/glove/).

However, we might want to hold off a bit before working on this, given [the discussion over this proposal](https://github.com/src-d/guide/pull/163).
